### PR TITLE
Adding Otel Operator v0.52.0

### LINKED ISSUE #890: Release v0.52.0
See https://github.com/open-telemetry/opentelemetry-collector/issues/5415

### PR DESCRIPTION
**Pull request by yuriolisa**
Signed-off-by: Yuri Sa <yurimsa@gmail.com>

Fixes #890

### CODE REVIEW

**Comment by yuriolisa** [comment]
Resolve #890

**Comment by yuriolisa** [comment]
@pavolloffay, @VineethReddy02

File: README.md [hunk 1]
```diff
@@ -297,6 +297,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.52.0                | v1.19 to v1.23       | v1                   |
```
Comment (pavolloffay) [inline README.md:300]: Is the cert-manager v1 here because we updated to 1.18?

**Review by pavolloffay — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yuriolisa) [reply README.md:300]: Yes.

**Comment by yuriolisa** [comment]
@pavolloffay, are you able to merge it?

**Pull request merged**